### PR TITLE
[NETBEANS-54] Module Review hibernate

### PR DESCRIPTION
**Pull request by ikysil**
- external binary contains DTDs for Hibernate 3.0 configuration files
- changed license headers for files reported by Rat
- replacing the bundled binary is tracked by https://issues.apache.org/jira/browse/NETBEANS-79

### CODE REVIEW

File: hibernate/src/org/netbeans/modules/hibernate/resources/HibernateRevengTemplate.xml [hunk 1]
```diff
@@ -1,2 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (matthiasblaesing) [inline hibernate/src/org/netbeans/modules/hibernate/resources/HibernateRevengTemplate.xml]: From the name this looks to be a template for a user file. If so, I would not include an apache license here, if it ends up int he users code base. There is no real IP here, so I'd be just annoyed if I'd have to remove/replace the license by hand.

Comment (ikysil) [reply hibernate/src/org/netbeans/modules/hibernate/resources/HibernateRevengTemplate.xml]: Okay, I see.

**Comment by ikysil** [comment]
@matthiasblaesing Could you please do another round of review?

**Comment by matthiasblaesing** [comment]
Looks better, but you still introduce a change into HibernateRevengTemplate.xml. You add a XML prolog to the file, that was not there before.

So could you please:
- remove that also
- rebase your changes onto master
- squash your changes together into one changeset

Thank you!

**Comment by ikysil** [comment]
It looks like this particular pull request is broken beyond reasonable repair due to mistakes which I did during synchronization with upstream. I will open a new PR.

**Pull request closed without merging**